src/components/Inputs/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Search from '../../../icons/regular/Search';
+import Cross from '../../../icons/regular/Cross';
+import BaseInput from '../BaseInput';
+import { ColorsMap, GeneralProps } from '../types';
+
+interface SearchInputSpecificProps {
+  onClear: () => void;
+}
+
+type SearchInputProps = SearchInputSpecificProps & GeneralProps;
+
+const SearchInput = ({
+  size = 'large',
+  placeholder,
+  state = 'idle',
+  value,
+  onChange,
+  onClear,
+}: SearchInputProps) => {
+  const leftIconSize = size === 'large' ? 'w-4 h-4' : 'w-3.5 h-3.5';
+  const rightIconSize = size === 'large' ? 'w-3 h-3' : 'w-2.5 h-2.5';
+
+  const hasValue = value !== '';
+  const notDisabled = state !== 'disabled';
+  const showClearButton = hasValue && notDisabled;
+
+  const idleColor = value ? 'border-dark-1000' : 'border-dark-700';
+
+  const colors: ColorsMap = {
+    idle: `${idleColor}`,
```

Comment:
Is this ok? Idle is a border color instead of a text-color? Why do we have different text-green and then border-dark